### PR TITLE
Eliminate some compilation warnings (#379)

### PR DESCRIPTION
**Pull request by jlaine**
I'd like to fix a couple more warnings before merging.

### CODE REVIEW

File: av/video/format.pyx [hunk 1]
```diff
@@ -128,7 +128,7 @@ cdef class VideoFormatComponent(object):
     property bits:
         """Number of bits in the component."""
         def __get__(self):
-            return self.ptr.depth_minus1 + 1
+            return self.ptr.depth
```
Comment (mikeboers) [inline av/video/format.pyx]: Ahaha what a change.

Comment (jlaine) [reply av/video/format.pyx]: I'm really good at math huh? ;)

Comment (jlaine) [reply av/video/format.pyx]: Ah I forgot to remove the comment referring to a bitfield, these are just plain ints

**Comment by jlaine** [comment]
Concerning *_register_all the deprecation only starts with ffmpeg 4.0. i need to check whether they were already no-oops for 3.x (in which case we can drop them) or whether we need some conditional logic.

**Comment by jlaine** [comment]
I had closer look at audio_rates / frame_rates, I'll move the discussiion over to a separate PR

**Comment by jlaine** [comment]
If the rest of the changes look good, can get this first round merged?

**Comment by mikeboers** [comment]
Yeah. I'll do it in a few hours. 😁

**Pull request merged**